lib/slack/bot.ex

```diff
@@ -176,6 +194,10 @@ defmodule Slack.Bot do
 
   def websocket_handle(_, _conn, state), do: {:ok, state}
 
+  defp bot_to_map(bot) do
+    %{"#{bot.id}" => bot}
```

Comment:
The tests pass for my original code and fail for your suggestion. Using what I think you want to use, `Map.get(bot, "id")` will result in `nil => %{id: "123"}` for the tests instead of the appropriate `"123" => %{id: "123"}` - what version of Elixir are you using? If you're on the supported version these tests should pass for you

Reply:
Tests are passing, it's use against real API that's failing. #114 describes the reason for it.

Basically, RTM and Bot modules decode JSON with `keys: :atoms` option. Whereas [Slack.Web.DefaultClient](https://github.com/BlakeWilliams/Elixir-Slack/blob/2a3739d13fe1ff41ae8a03356efaa7e61e4778ca/lib/slack/web/default_client.ex#L25) doesn't.

Generally speaking, it might be a good idea to get rid of keys to atoms conversion as a part of major version release.

Reply:
> Generally speaking, it might be a good idea to get rid of keys to atoms conversion as a part of major version release.

Agreed, that was a design decision made before the library was a library and it kinda snuck in. That'd be a great issue if there isn't one already.